frontend/app/services/address-service.server.ts

```diff
@@ -37,11 +37,14 @@ function createAddressService() {
   const { INTEROP_API_BASE_URI } = getEnv();
 
   async function getAddressInfo(userId: string, addressId: string) {
+    log.trace('Calling getAddressInfo');
```

Comment:
I think this should be logged at the DEBUG level, and should contain the user id and address ID.. something like:

```
log.debug('Fetcing address info [%s] for user [%s]', addressId, userId);
```

Think about when we're looking through the logs, trying to troubleshoot an address issue. Seeing "calling getaddressinfo" in the logs with no more information isn't useful.

Reply:
Also, we generally wrap the values in some delimiter (I use `[..]`), so we can see if there are any extra spaces or other weird characters.

Reply:
Fair point. I've made changes accordingly.